brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
[Microsoft is soon dropping the support for old IEs](https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support), so it may be one of the reasons to drop it here too